### PR TITLE
cellvar working?

### PR DESCRIPTION
**Pull request by Farnazmdi**
Dr. Meyer, could you please see if this is correct? Nothing fails and I checked in the terminal that the functions in the class are working and nothing is breaking. If it is correct, JC and I can do this for the rest of the files.
Thank you.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -15,3 +15,4 @@ jax==0.2.19
 jaxlib==0.1.70
 networkx==2.6.2
 statsmodels==0.12.2
+typing
```
Comment (aarmey) [inline requirements.txt:18]: `typing` is built into Python. There is no module.

File: lineage/CellVar.py [hunk 2]
```diff
@@ -150,14 +152,14 @@ def get_subtrees(node, lineage):
     return subtree, not_subtree
 
 
-def find_two_subtrees(cell, lineage):
+def find_two_subtrees(Cell: cell, lineage: list):
```
Comment (aarmey) [inline lineage/CellVar.py:155]: I don't like having a type and variable of essentially the same name. Why not call the type `cellType`, and the argument `cell` like before?

File: lineage/CellVar.py [hunk 3]
```diff
@@ -32,7 +34,7 @@ def __init__(self, parent, gen, **kwargs):
             self.right = kwargs.get("right", None)
             self.obs = kwargs.get("obs", [])
 
-    def divide(self, T):
+    def divide(self, T: np.ndarray):
```
Comment (aarmey) [inline lineage/CellVar.py:37]:
```suggestion
    def divide(self, T: np.ndarray[float]):
```

**Review by aarmey — CHANGES_REQUESTED** [review]
Close! A couple edits.

File: lineage/CellVar.py (hunk 3 above)
Comment (Farnazmdi) [reply lineage/CellVar.py]: It says `Type object is not subscriptable`..

Comment (aarmey) [reply lineage/CellVar.py]: Ok, ignore that then.

**Comment by Farnazmdi** [comment]
I am not authorized to merge. Could you please merge?

**Pull request merged**